newIDE/app/src/AssetStore/index.js

```diff
@@ -483,8 +477,8 @@ export const AssetStore = React.forwardRef<Props, AssetStoreInterface>(
             tooltip={t`Back to discover`}
             onClick={() => {
               setSearchText('');
-              navigationState.openHome();
-              setScrollUpdateIsNeeded();
+              const page = navigationState.openHome();
+              setScrollUpdateIsNeeded(page);
```

Comment:
Do you mean that is code won't work?
```suggestion
              navigationState.openHome();
              const page = getCurrentPage();
              setScrollUpdateIsNeeded(page);
```

Reply:
Oh maybe this will work! I'll try it right away

Reply:
It doesn't work 😬 Sometimes this is a bit hard to understand

Reply:
oh wow ok.